### PR TITLE
Describe privacy inplications of changing payment method

### PR DESCRIPTION
**Pull request by marcoscaceres**
Attempt to address feedback from @wseltzer as part of PING feedback. 

The following tasks have been completed:

 * [X] Confirmed there are no ReSpec errors/warnings.
 * [ ] Has undergone security/privacy review 
 
Optional, impact on Payment Handler spec?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/pull/849.html" title="Last updated on Mar 13, 2019, 2:09 AM UTC (335770d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/849/2b23c2f...335770d.html" title="Last updated on Mar 13, 2019, 2:09 AM UTC (335770d)">Diff</a>

### CODE REVIEW

**Review by rsolomakhin — APPROVED** [review]
Looks good from the standpoint of payment handlers. The PH API does not yet have the payment method change event defined, but I will keep this PR in mind when adding it.

File: index.html [hunk 1]
```diff
@@ -222,10 +222,21 @@ <h2>
...
-          <a data-cite="WEBIDL#idl-object">object</a> or null.
+          <p>
+            Steps that describe how to handle the user changing payment method
+            or monetary instrument (e.g., from a debit card to a credit card)
+            that results in a <a data-cite=
+            "WEBIDL#idl-dictionary">dictionary</a> or <a data-cite=
+            "WEBIDL#idl-object">object</a> or null.
+          </p>
+          <p>
+            A <a>payment handler</a> that define <a>steps for when a user
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
            A <a>payment handler</a> that defines <a>steps for when a user
```

File: index.html [hunk 2]
```diff
@@ -5300,6 +5320,13 @@ <h2>
           member with a third-party <a>payment handler</a> without user
           consent.
         </p>
+        <p>
+          Where sharing of privacy sensitive information might not be obvious
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          Where sharing of privacy-sensitive information might not be obvious
```

File: index.html [hunk 3]
```diff
@@ -5300,6 +5320,13 @@ <h2>
...
+          to users (e.g., <a data-lt=
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          to users (e.g., when <a data-lt=
```

File: index.html [hunk 4]
```diff
@@ -5300,6 +5320,13 @@ <h2>
...
+          "payment method changed algorithm">changing payment methods</a>), it
+          is RECOMMEDED that user agents inform user of exactly what
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          is RECOMMENDED that user agents inform the user of exactly what
```

**Review by ianbjacobs — CHANGES_REQUESTED** [review]
Hi Marcos, 

Minor editorial suggestions.

**Review by ianbjacobs — APPROVED** [review]

**Pull request merged**